test/ex_ftx/account/show_test.exs

```diff
@@ -10,11 +10,13 @@ defmodule ExFtx.Account.ShowTest do
 
   @valid_credentials %ExFtx.Credentials{
     api_key: System.get_env("FTX_API_KEY"),
-    api_secret: System.get_env("FTX_API_SECRET")
+    api_secret: System.get_env("FTX_API_SECRET"),
```

Comment:
I feel like we should now make this test all 3 account permutations. e.g.

```elixir
@valid_main_account_credentials ...
@valid_sub_account_credentials ...
@invalid_credentials ...
```

Reply:
A bit conflicted on this - fundamentally, sub account credentials would do everything the main account ones can, except they only work when the FTX-SUBACCOUNT header is present. Main account credentials will default to the main account if no subaccount is specified, but also work for subaccounts if it is. We would have to do 5 tests then to validate that all of that works as expected:

1. Unrestricted credentials, nil subaccount -> Should work
2. Unrestricted credentials, with subaccount -> Should work
3. Subaccount credentials, nil subaccount -> Fail
4. Subaccount credentials, with subaccount -> Should work
5. Invalid credentials -> Fail

If that sounds reasonable, I can update the PR to match

Reply:
I just want to do that test of all permutations for 1 endpoint. It could even be a new test.